File: Java/Pattern1.java
import java.util.Scanner;

public class Pattern1 {
    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter the number: ");
        int num = scan.nextInt();
        scan.close();

        for(int i=1; i<=num; i++) {
            for(int j=1; j<=i; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: C/pattern1.c
#include <stdio.h>

int main() {
    int num, times;
    printf("Enter number for pattern: ");
    scanf("%d", &num);
    printf("Enter number for times: ");
    scanf("%d", &times);

    for(int i=0; i<=num-1; i++) {
        for(int x=1; x<=times; x++) {
            for(int j=1; j<=(num-i); j++){
                printf("*");
            }
            for(int j=1; j<=(2*i); j++) {
                printf(" ");
            }
            for(int j=1; j<=(num-i); j++) {
                printf("*");
            }
        }
        printf("\n");
    }
    for(int i=2; i<=num; i++){
        for(int x=1; x<=times; x++) {
            for(int j=1; j<=i; j++) {
                printf("*");
            }
            for(int j=1; j<=(2*(num-i)); j++) {
                printf(" ");
            }
            for(int j=1; j<=i; j++) {
                printf("*");
            }
        }
        printf("\n");
    }
}